utils/src/main/kotlin/OkHttpClientHelper.kt

```diff
@@ -109,6 +111,31 @@ object OkHttpClientHelper {
      */
     suspend fun await(request: Request, block: BuilderConfiguration? = null): Response =
         buildClient(block).newCall(request).await()
+
+    /**
+     * Download from [url] and return a [Result] with a string representing the response body content on success, or a
+     * [Result] wrapping a [HttpDownloadError] on failure.
+     */
+    fun downloadText(url: String): Result<String> {
+        val request = Request.Builder().get().url(url).build()
+
+        return execute(request).use { response ->
```

Comment:
`execute()` can thrown an `IOException`. As `downloadText()` returns a `Result`, I think, this exception should be caught and wrapped inside a failure result. This, however, clashes with `HttpDownloadError` in its current form, because in this case no status code is available. So the error class probably has to be extended somehow.

Reply:
Good point. I'll think about addressing this in a follow-up.

Reply:
See https://github.com/oss-review-toolkit/ort/pull/3890.